sass/blocks/wrapper/_wrapper.scss

```diff
@@ -0,0 +1,12 @@
+//Desktop screen
+.wrapper {
```

Comment:
I am a little bit confused. Why does this PR contain code for the wrapper and the other one contains code for nav media-query. I wasn't expecting that. Looks like two duplicate PRS.